test/hotjar.browser.js

```diff
@@ -68,121 +51,54 @@ test('test disclousure is suppressing name', async t => {
-      manuscript.supportingFiles[0],
-    ])
-    .expect(files.supportingFile.count)
-    .eql(1)
-    .click(files.supportingFilesRemove)
-    .setFilesToUpload(files.supportingFilesUpload, manuscript.supportingFiles)
-    .expect(files.supportingFile.count)
-    .eql(2)
-    .click(wizardStep.next)
-
-  // adding manuscript metadata
-  await t
-    .expect(submission.title.value)
-    .eql(manuscript.title)
-    .click(submission.articleType)
-    .click(submission.articleTypes.nth(0))
-    .click(submission.subjectAreaLabel)
-    .pressKey('enter')
-    .pressKey('down')
-    .pressKey('enter')
-    .click(submission.discussionCheckbox)
-    .typeText(submission.discussionText, 'Spoke to Bob about another article')
-    .click(submission.previousArticleCheckbox)
-    .typeText(submission.previousArticleText, 'A title')
-    .click(submission.cosubmissionCheckbox)
-    .typeText(submission.firstCosubmissionTitle, 'Another title')
-    .click(submission.moreSubmission)
-    .typeText(submission.secondCosubmissionTitle, 'Yet another title')
-    .click(wizardStep.next)
-
-  // selecting suggested and excluded editors & reviewers
   await t
-    .click(editors.suggestedSeniorEditorSelection)
-    .click(editors.peoplePickerOptions.nth(0))
-    .click(editors.peoplePickerOptions.nth(2))
-    .click(editors.peoplePickerOptions.nth(3))
-    .click(editors.peoplePickerOptions.nth(5))
-    .click(editors.peoplePickerOptions.nth(7))
-    .click(editors.peoplePickerOptions.nth(9))
-    .click(editors.peoplePickerSubmit)
-    .click(editors.suggestedReviewingEditorSelection)
-    .click(editors.peoplePickerOptions.nth(1))
-    .click(editors.peoplePickerOptions.nth(4))
-    .click(editors.peoplePickerOptions.nth(6))
-    .click(editors.peoplePickerOptions.nth(8))
-    .click(editors.peoplePickerOptions.nth(10))
-    .click(editors.peoplePickerOptions.nth(11))
-    .click(editors.peoplePickerSubmit)
-    .expect(editors.validationErrors.withText(/./).count)
-    .eql(0)
     .typeText(editors.firstReviewerName, 'Edward')
     .typeText(editors.firstReviewerEmail, 'edward@example.com')
     .typeText(editors.secondReviewerName, 'Frances')
     .typeText(editors.secondReviewerEmail, 'frances@example.net')
     .typeText(editors.thirdReviewerName, 'George')
     .typeText(editors.thirdReviewerEmail, 'george@example.org')
-    .typeText(editors.fourthReviewerName, 'Ayesha')
-    .typeText(editors.fourthReviewerEmail, 'ayesha@example.com')
-    .typeText(editors.fifthReviewerName, 'Sneha')
-    .typeText(editors.fifthReviewerEmail, 'sneha@example.net')
-    .typeText(editors.sixthReviewerName, 'Emily')
-    .typeText(editors.sixthReviewerEmail, 'emily@example.org')
-    .click(wizardStep.next)
-
-  // consenting to data disclosure
+  navigationHelper.navigateForward()
+
+  // disclosure's page
   await t
-    .typeText(disclosure.submitterName, 'Joe Bloggs')
     .expect(Selector('[data-test-id=disclosure-name]'), {
```

Comment:
My be worth creating a `pageObject` for keeping track of hotjar omitted field selectors?

Reply:
done